packages/devextreme/js/__internal/grids/grid_core/editor_factory/m_editor_factory.ts

```diff
@@ -103,7 +115,11 @@ export class EditorFactory extends ViewControllerWithMixin {
   }
 
   _needHideBorder($element) {
-    return $element.hasClass(EDITOR_INLINE_BLOCK);
+    const rowsViewElement = this.getView('rowsView').element();
+    const isRowsView = $element.closest(rowsViewElement).length > 0;
```

Comment:
Question

The ```closest``` is a very expensive operation.
Can we just check the existence of the ```rowsView``` element here?
Or the ```rowsView``` may have the ```element``` when it's not attached to the DOM?

Something like this:
```typescript
const isRowsView = !!this.getView('rowsView').element?.();
```

Reply:
We found that closest is appropriate operation because it traversing up through its ancestors in the DOM tree https://api.jquery.com/closest/

Reply:
I was wrong, the ```closest``` traverse DOM tree only up and match parent nodes.
So, using ```closest``` is ok :)